### PR TITLE
fix: updated redis library

### LINKED ISSUE #99: Upgrade dependency `gopkg.in/redis.v4` to `github.com/redis/go-redis/v9`
Even Dependabot does not understand what to do with this version of this dependency ^^
Screenshot 2023-02-03 at 10-35-00 Update details · Scalingo_go-philae

### PR DESCRIPTION
**Pull request by yohann-bacha**
fixes #99

### CODE REVIEW

**Review by EtienneM — COMMENTED** [review]
Just a question, otherwise LGTM :)

File: go.mod [hunk 1]
```diff
@@ -11,17 +11,18 @@ require (
...
 )
 
 require (
+	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	golang.org/x/term v0.5.0 // indirect
-	golang.org/x/text v0.7.0 // indirect
 )
 
 require (
```
[the remark this answers is not in the document]
Comment (yohann-bacha) [reply go.mod:28]: Weird that go mod tidy did that, fixing

**Comment by yohann-bacha** [comment]
This is gonna be complicated to merge if dependabot continues to auto merge

**Review by EtienneM — APPROVED** [review]

**Comment by yohann-bacha** [comment]
dependabot made this too hard to merge, made a new branch from master here https://github.com/Scalingo/go-philae/pull/135

**Pull request closed without merging**